gf-guide/metrics.md

```diff
@@ -248,26 +248,85 @@ If the font is already hosted on [fonts.google.com](http://fonts.google.com/), y
 
 ## CJK Vertical Metrics
 
-CJK vertical metrics are based on Source Han Sans and Noto CJK fonts.
+Vertical metrics for CJK fonts are based on the font emBox values. This ensures that the ideographs (or syllables) are properly positioned in the center of the emBox both in horizontal and vertical typesetting. 
 
-The following vertical metric values must be applied to all CJK fonts
+The following metrics are a distinct split from the standard approach of setting vertical metrics for CJK fonts, which normally set the `sTypo` metrics to align with the emBox values. This is also how the [OT spec recommendations](https://learn.microsoft.com/en-us/typography/opentype/spec/os2#stypoascender) are written. However, following [investigation into performance of CJK fonts](https://github.com/google/fonts/issues/8911) under the primary scenarios that Google Fonts prioritizes, the following new metrics have been established:
 
-| Attrib                                    | Value                                    | Example using 1000upm font |
+| Attrib                                    | Value                                    | Example using [Iansui](https://github.com/ButTaiwan/iansui)       |
```

Comment:
The original (1000upm) should be retained as other UPM values are still valid for C/J fonts too.

Reply:
I did think it was helpful to link to the specific font here (since the metrics are font-specific), but agree that being clearer about the UPM would probably also be useful.